platform/linux-generic/odp_ipsec.c

```diff
@@ -682,18 +720,18 @@ static ipsec_sa_t *ipsec_in_single(odp_packet_t pkt,
 		else
 			status->error.alg = 1;
 
-		goto err;
+		goto exit;
 	}
 
 	if (_odp_ipsec_sa_replay_update(ipsec_sa,
 					state.in.seq_no,
 					status) < 0)
-		goto err;
+		goto exit;
 
-	if (_odp_ipsec_sa_stats_update(ipsec_sa,
+	if (_odp_ipsec_sa_lifetime_update(ipsec_sa,
```

Comment:
Indentation of the unchanged lines should be fixed. Checkpatch also complains about it.

Reply:
Missed this one during rebase. Will be fixed in next version.